src/Elliptic/Systems/GetSourcesComputer.hpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+namespace elliptic {
+namespace detail {
+template <typename System, typename = std::void_t<>>
+struct sources_computer_linearized {
+  using type = typename System::sources_computer;
+};
+template <typename System>
+struct sources_computer_linearized<
+    System, std::void_t<typename System::sources_computer_linearized>> {
+  using type = typename System::sources_computer_linearized;
+};
+}  // namespace detail
+
+template <typename System, bool Linearized>
+using get_sources_computer = tmpl::conditional_t<
```

Comment:
should this have some Doxygen documentation?

Reply:
Sure, that was an oversight on my part. I squashed that in directly. Thanks!